tests/arc/torc_basic_test.nim

```diff
@@ -136,3 +139,5 @@ main()
 main2()
 GC_fullCollect()
 echo "MEM ", getOccupiedMem()
+
+echo defined(gcArc)
```

Comment:
`doAssert defined(gcArc)`

https://nim-lang.github.io/Nim/contributing.html

> Use boolean conditions and doAssert for the testing by itself, don't rely on echo statements or similar;

(even though in the past we've relied on echo in tests)

Reply:
Done :)